recipes/server.rb

```diff
@@ -24,7 +24,7 @@
 node.default['percona']['conf']['mysqld']['log_bin_trust_function_creators'] = '1'
 
 # utf8mb4 support
-node.default['percona']['conf']['mysqld']['innodb_large_prefix'] = 'true'
+node.default['percona']['conf']['mysqld']['innodb_large_prefix'] = 'true' if node['platform_version'].to_i < 8
```

Comment:
Does this not work with the newer Percona 8.0?

Reply:
This option was deprecated in Percona 5.7 and removed in 8.0 https://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_large_prefix

Reply:
Instead of doing this based off of the platform version, you should do it based on the ``node['percona']['version']`` attribute.